api/v1/utils/roles.js

```diff
@@ -0,0 +1,10 @@
+var _ = require('lodash');
+
+var ALL_ROLES = ['ADMIN', 'STAFF', 'SPONSOR', 'MENTOR', 'VOLUNTEER', 'HACKER'];
```

Comment:
Yeah I was planning on discussing when I got to #3 and/or #7. Until then this is really only in there as a placeholder, so anyone's free to let me know how the role-structure should go.